examples/solverdummies/precice-config.xml

```diff
@@ -51,8 +51,8 @@
     <participants first="SolverOne" second="SolverTwo" />
     <max-time-windows value="2" />
     <time-window-size value="1.0" />
+    <min-iterations value="2" />
     <max-iterations value="2" />
-    <min-iteration-convergence-measure min-iterations="5" data="Data-One" mesh="SolverOne-Mesh" />
```

Comment:
Why was this 5 min-iterations before? Looks like it was a bit buggy before. Intuitively, I would expect only min iterations in this simple case where we want to say "do exactly X iterations". But this contradicts a bit what we already discussed, right?

Reply:
Maybe we really should implement some actual super simple functionality here.